spec/services/finance/ecf/output_calculator_spec.rb

```diff
@@ -30,22 +30,43 @@
   describe "#fee_for_declaration" do
     subject { first_statement_calc }
 
-    it do
+    it "returns correct fees" do
       expect(subject.fee_for_declaration(band_letter: :a, type: :started)).to eql(48)
       expect(subject.fee_for_declaration(band_letter: :a, type: :retained_1)).to eql(36)
+      expect(subject.fee_for_declaration(band_letter: :a, type: :retained_2)).to eql(36)
+      expect(subject.fee_for_declaration(band_letter: :a, type: :retained_3)).to eql(36)
+      expect(subject.fee_for_declaration(band_letter: :a, type: :retained_4)).to eql(36)
       expect(subject.fee_for_declaration(band_letter: :a, type: :completed)).to eql(48)
+      expect(subject.fee_for_declaration(band_letter: :a, type: :extended_1)).to eql(36)
+      expect(subject.fee_for_declaration(band_letter: :a, type: :extended_2)).to eql(36)
+      expect(subject.fee_for_declaration(band_letter: :a, type: :extended_3)).to eql(36)
 
       expect(subject.fee_for_declaration(band_letter: :b, type: :started)).to eql(36)
+      expect(subject.fee_for_declaration(band_letter: :b, type: :retained_1)).to eql(27)
       expect(subject.fee_for_declaration(band_letter: :b, type: :retained_2)).to eql(27)
+      expect(subject.fee_for_declaration(band_letter: :b, type: :retained_3)).to eql(27)
+      expect(subject.fee_for_declaration(band_letter: :b, type: :retained_4)).to eql(27)
       expect(subject.fee_for_declaration(band_letter: :b, type: :completed)).to eql(36)
```

Comment:
maybe we can add extended to band b as well?

Reply:
Updated